script/metalsmith/breadcrumbTransformer.js

```diff
@@ -0,0 +1,26 @@
+const cheerio = require('cheerio');
+const fs = require('fs');
+const path = require('path');
+const UglifyJS = require('uglify-js');
+
+const breadcrumb = fs.readFileSync(path.join(__dirname, '../../assets/js/breadcrumbs/mobile-breadcrumb.js'), 'utf8');
```

Comment:
You definitely do not need to do this, but we could also probably run this file through Babel's API here. Maybe something to keep in mind in the future, so we can write more modern JS.

Reply:
I gave this a spin, and wasn't able to get Babel and UglifyJS working. There's something about how UglifyJS expects the file entry that `babel.transform` goofs up, and I wasn't able to find the right approach. I'll keep it in the backlog of nice to haves, but per our Slack discussion, uglifying is the more important item for first release.